forch/forchestrator.py

```diff
@@ -85,7 +85,18 @@ def initialize(self):
         self._cpn_collector.initialize()
         LOGGER.info('Using peer controller %s', self._get_peer_controller_url())
         self._register_handlers()
+        self._create_authenticator()
 
+        device_info = self._config.get('static_device_info', {})
+        if 'static_device_placement' in device_info:
+            placement_file = os.path.join(
+                os.getenv('FAUCET_CONFIG_DIR'), device_info['static_device_placement'])
+            device_placement_info = yaml_proto(placement_file, DevicesState).device_mac_placements
+            for eth_src, device_placement in device_placement_info.items():
+                self.process_device_placement(eth_src, device_placement)
+        self._initialized = True
+
+    def _create_authenticator(self):
```

Comment:
this doesn't actually create the authenticator, it *maybe* creates authenticator. So, either
1) call it _maybe_create_authenticator()
2) only call _create_authenticator() if 'radius_info' is defined (and then pass in radius_info itself)